mtz-auth-interceptor.html

```diff
@@ -41,7 +41,7 @@
        * @param {String} eventName
        */
       intercept(event, eventName) {
-        if (event.target.getAttribute('with-auth'))
+        if (event.target.getAttribute('with-auth') === '')
```

Comment:
`with-auth`
`with-auth=""`
and
`with-auth="WiTh-auTh"`
are all true according to the html spec, so either `!== null` or do a case insensitive match against `with-auth` for the value.